content/docs/user-guide/networking/multiplayer/_index.md

```diff
@@ -26,6 +26,7 @@ The Multiplayer Gem supports the following:
 | [Configuring a Project](configuration) | How to add and enable the O3DE Multiplayer Gem in a project. |
 | [Running Multiplayer Projects](running) | How to run projects that use the O3DE Multiplayer Gem. |
 | [Multiplayer Auto-components](autocomponents) | How to automatically create components for use with the Multiplayer Gem using the AzAutoGen system. |
+| [Separating Client and Server](split) | How to separate client and server logic and build dependencies.
```

Comment:
Minor: split is more an internal short-name. Maybe this should be code_separation? Or launchers? so the page has a more meaningful url?

Reply:
I can rename to code_separation.